packages/exporter-trace-otlp-grpc/src/util.ts

```diff
@@ -84,10 +84,12 @@ export function send<ExportItem, ServiceRequest>(
 ): void {
   if (collector.serviceClient) {
     const serviceRequest = collector.convert(objects);
+    const deadline = new Date().setSeconds(new Date().getSeconds() + (collector.timeoutMillis / 1000));
```

Comment:
Nit: can we calculate the deadline using unix-epoch instead of using the implicit-coercing setter of Date? gRPC client supports milliseconds as the deadline value: https://grpc.github.io/grpc/node/grpc.html#~Deadline